packages/xo-server-backup-reports/src/index.js

```diff
@@ -117,6 +118,19 @@ const createGetTemporalDataMarkdown = formatDate => (
   return markdown
 }
 
+const addWarnings = (text, warnings, nbIndent = 0) => {
+  if (warnings === undefined) {
+    return
+  }
+
+  const indent = INDENT.repeat(nbIndent)
+  const warningsText = warnings
+    .map(({ message }) => `- **${ICON_INTERRUPTED} ${message}**`)
```

Comment:
I think it would be clearer (and more efficient) to simply do:

```js
warnings.forEach(({ message }) => {
  text.push(`${indent}- **${ICON_WARNING} ${message}**)
})
```